contracts/container/config.yml

```diff
@@ -9,7 +9,7 @@ events:
       - name: containerID
         type: Hash256
       - name: publicKey
```

Comment:
Then this is no longer a `publicKey`, rather an `owner`.

Reply:
kept the old event